ports/air-ctl/portfile.cmake

```
@@ -1,10 +1,8 @@
-vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
-
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO  inie0722/CTL
     REF v1.0.0
```

```suggestion
    REF 58ca0f229aff093ada66955993cada93e01da18c #1.0.0
```
Sorry, this is my mistake, this suggestion should be here. If you will create new tags in future updates, the tag is ok. Due to the same tag has been used in previous updates, it will cause baseline issue.

---

And I suggest this PR use commit ID, CI build failed with the following error:
```
Installing 3/3 air-ctl:x64-windows...
Building air-ctl[core]:x64-windows...
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:14 (message):
  

  File does not have expected hash:

          File path: [ D:/downloads/inie0722-CTL-v1.0.0.tar.gz ]
      Expected hash: [ 6e1fcd70be0750b3e45de80d6fb30471fdbd8bbd23afd7ff9fb8c3b9fe61669cd02773cf2b6573348dd2905a303845627d3f5f3e647f4e1261db4cc570cf5099 ]
        Actual hash: [ facd0afdb20437c0ffa5b097a68e2028faa314be6052e0db1e2c727ce5a0305bca8e575c946b0e10a6343dfaddc38770c1e5a56ae201e1aa906883df6d662398 ]

  The cached file SHA512 doesn't match.  The file may have been corrupted.
  To re-download this file please delete cached file at path
  D:/downloads/inie0722-CTL-v1.0.0.tar.gz and retry.
```
The reason should be cache, if you release a new version with a new tag, you can use tag on next update.

---

Understood, I have modified